tsconfig.json

```diff
@@ -4,18 +4,18 @@
 
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "ESNext",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
-    "module": "ESNext",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "target": "ES5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Something is going wrong here. Get dozens of error messages like this and it doesn't build. 
`Transforming const to the configured target environment ("ES5") is not supported yet`

Changing it to ES3 solves allows it to build but it blackscreens on Mocha. Though it runs on modern Chrome, it's hitting other issues in the console. 
`main.js:1 Uncaught (in promise) DOMException: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The HTMLImageElement provided is in the 'broken' state.`

Also do we know if **_es2015_** module setup  on line 8 is compatible with Mocha? I suspect possibly not as it's 3 years after Opera 12 came out  (2012)